src/internal/modules/charge-information/forms/charge-element/quantities.js

```diff
@@ -11,17 +11,17 @@ const getFormField = (key, data) => {
   return fields.text(fieldName, {
     controlClass: 'govuk-input govuk-input--width-10',
     label: `${capitalize(key)}${key === 'billable' ? ' (optional)' : ''}`,
+    type: 'number',
     suffix: 'megalitres per year',
-    mapper: 'numberMapper',
     errors: {
-      'any.empty': {
+      'any.required': {
         message: `Enter an ${key} quantity`
       },
-      'number.base': {
-        message: `Enter a valid ${key} quantity as a number that is more than zero`
+      'any.empty': {
+        message: `Enter an ${key} quantity`
       },
-      'number.greater': {
-        message: `Enter a valid ${key} quantity as a number that is more than zero`
+      'string.regex.base': {
+        message: `Enter a number for the ${key} quantity using 6 decimal places or fewer, the number must be more than 0`
```

Comment:
May I suggest instead of `the number must be more than 0` saying `the quantity must be greater than 0`?

Again, pedantry. So feel free to ignore this

Reply:
This came out of a chat with UCD, so I'd rather not change it